README.md

```diff
@@ -1 +1,184 @@
+- *[Webpack Configuration Setup]*
+- *[Follow HTML and CSS best practice]*
+- *[Display List]*
+ 
+
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+## :rocket: Live Demo <a name="live-demo"></a>
+ [comming soon]
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+## 💻 Getting Started <a name="getting-started"></a>
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+In order to run this project you need:
+- GitHub Account.
+- Git bash.
+- Visual Studio Code.
+- Node JS.
+
+### Setup
+Clone this repository to your desired folder:
+Use git clone command or download ZIP folder
+sh
+  cd folder Name 
+git@github.com:isume295/My_Portfolio.git
+
+### Install
+Install this project with:
+npm
+sh
+  cd project Name
+  npm init -y
+
+### Usage
+To run the project, execute the following command:
+npm start or live server
+
+sh
+  GitHub Pages Server
+
+### Run tests
+To run tests, run the following command:
+ 
+ to test validation errors
+ npx hint . 
+ 
+ to test linter Errors
+ npx stylelint "**/*.{css,scss}"
+
+### Deployment
+You can deploy this project using:
+GitHub Pages
+
+[comming soon]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+## :busts_in_silhouette: Authors <a name="authors"></a>
+
+:bust_in_silhouette: *Sumeya Ibrahim*
+
+- GitHub: [@isume295](https://github.com/isume295/)
+- Twitter: [@isume295](https://twitter.com/isume295)
+- LinkedIn: [@isume295](https://www.linkedin.com/in/sumeya-ibrahim-109002232)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+## :telescope: Future Features <a name="future-features"></a>
+
+- [ ] *[Animations]*
+- [ ] *[Add List]*
+- [ ] *[Remove List]*
+- [ ] *[Remove completed list]*
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/isume295/To_Do_List/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project I would like to inform you that this project will improve and add other exciting features in the next release. stay tuned!!!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for providing the necessary tools to complete this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- walkthrough -->
+
+##  Walkthrough <a name="walkthrough"></a>
+
+ [comming soon]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- LICENSE -->
+
+## :memo: License <a name="license"></a>
+This project is [MIT]  licensed.
```

Comment:
- [OPTIONAL] I think your license file is not correctly linked here. I'll suggest fixing this before merging this pull request as it will make your readme appear more professional. The correct code for this line should be `This project is [MIT](./LICENSE)  licensed.` :+1: :smile:

Reply:
okay, tnxs